src/Formatter/FormatPostMentions.php

```diff
@@ -34,6 +45,17 @@ public function __invoke(Renderer $renderer, $context, $xml, Request $request =
                 $attributes['displayname'] = $post->user->display_name;
             }
 
+            $attributes['deleted'] = false;
+
+            if (! $post) {
+                $attributes['displayname'] = $this->translator->trans('flarum-mentions.forum.post_mention.unknown_text');
```

Comment:
Can we add a comment saying that we use unknown text for deleted post, because deleted text is deleted user?

Or alternatively, maybe the key should also be `deleted_text`?